packages/react-components/react-color-picker/library/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@fluentui/react-color-picker-preview",
-  "version": "0.3.0",
+  "name": "@fluentui/react-color-picker",
+  "version": "9.0.0",
```

Comment:
this is wrong , It seem you changed this by hand ? the migration generator provides different output. this would trigger a 9.0.1 release (9.0.0 of this package wouldn't exist )